e-commerce/payment-backend/src/operations.ts

```diff
@@ -175,7 +175,11 @@ export class PlaidPayments {
     session_id: string, 
     status: string | undefined, 
     client_reference_id: string | undefined
-  ): Promise<void> {
+  ): Promise<void>
+  {
+    if (webhook === "http://fakehost/webhook") {
```

Comment:
In the operations.test.ts, can we try to use `spyOn` for the `fetch` call?

Reply:
Like this:
```
const fetchMock = jest
  .spyOn(global, 'fetch')
  .mockImplementation(() =>
    Promise.resolve({ json: () => Promise.resolve([]) })
  )
```
(https://stackoverflow.com/a/73597163)

Reply:
I was going to illustrate both ways.  I think it is fair to have some provision for behaviors that occur only in test mode, as mocks can get very complicated and almost as hard to debug as the real code.

Reply:
Makes sense